### ISSUE TITLE
yield-effects doesn't support destructuring

### ISSUE THREAD

**Issue by burabure** [pke/eslint-plugin-redux-saga#13]
the following syntax triggers a false positive on yield-effects

```js
const [foo, bar] = yield [
  call(PrintersAPI.fetchPrinters, optsWithCapabilities),
  call(PrintersAPI.fetchManualAddedPrinters, optsWithCapabilities),
  ]
```

I'll submit a fix during the weekend
